### PR TITLE
Bug fix with http method without param in HttpGatewayAcceptor

### LINKED ISSUE #513: Method not being called when making HTTP request to @ServiceMethod that have no input params
Method not being called when making HTTP request to @ServiceMethod that have no input params, response is empty and no error in log.

Example:

**Definition**
`@ServiceMethod("getVersion")
    Mono<String> getVersion();`

**Implementation**
`@Override
    public Mono<String> getVersion() {
        return Mono.just("version 1");
    }`

**Request:**

- POST

- URL: localhost:8080/adminservice/getVersion

- Headers: Content-Type:application/json

- Body: <empty>

### PR DESCRIPTION
**Pull request by stingion**
Fix #513

### CODE REVIEW

File: services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGatewayAcceptor.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
     return httpRequest
         .receive()
         .aggregate()
+        .switchIfEmpty(
+            Mono.defer(() -> ByteBufMono.just(ByteBufAllocator.DEFAULT.compositeBuffer())))
```
Comment (segabriel) [inline services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGatewayAcceptor.java]: what is composite buffer maybe we need an empty buffer?

**Review by ronenhamias — APPROVED** [review]
+1

File: services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGatewayAcceptor.java (hunk 1 above)
Comment (stingion) [reply services-gateway-http/src/main/java/io/scalecube/services/gateway/http/HttpGatewayAcceptor.java]: Changed

**Review by segabriel — APPROVED** [review]
:+1:

**Pull request merged**